cmd/commands/git-source.go

```diff
@@ -474,6 +480,11 @@ func RunGitSourceList(ctx context.Context, runtimeName string) error {
 
 	for _, gs := range gitSources {
 		name := gs.Metadata.Name
+		nameWithoutRuntimePrefix := strings.Replace(name, fmt.Sprintf("%s-", runtimeName), "", 1)
```

Comment:
strings.StripPrefix is a better fit

Reply:
I see that StripPrefix is part of net/http and not strings lib, am I missing something?